src/main/java/com/google/sps/servlets/NotifyStudent.java

```diff
@@ -98,6 +110,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendError(HttpServletResponse.SC_BAD_REQUEST);
     } catch (FirebaseAuthException e) {
       response.sendError(HttpServletResponse.SC_FORBIDDEN);
+    } catch (InterruptedException e) {
```

Comment:
You could use a [`multi-catch` block](https://stackoverflow.com/questions/3495926/can-i-catch-multiple-java-exceptions-in-the-same-catch-clause) if you do the same thing when catching multiple types of exceptions.